schemas/aws/elasticsearch-defaults-1.yml

```diff
@@ -102,5 +102,32 @@ properties:
         type: string
       indices.query.bool.max_clause_count:
         type: string
+  advanced_security_options:
+    type: object
+    additionalProperties: false
+    properties:
+      enabled:
+        type: boolean
+      internal_user_database_enabled:
+        type: boolean
+      master_user_options:
```

Comment:
Mmm... if everything related to this user is going to be in vault, do we need the extra level of indentation?

Reply:
I want to match the structure of the original terraform config as close as possible to make things easier... An in the future we may want to add the other auth option (via IAM) that AWS supports